src/network-config.js

```diff
@@ -143,6 +146,7 @@ export const networkConfigs = {
       chainId: 80001,
       testnet: true,
       ...KNOWN_CHAINS.get(80001),
+      fullName: 'Mumbai Testnet',
       live: true,
```

Comment:
Why don't we fix the use-wallet library to have the proper fullName and shortName instead of overwriting the value here in the app?